<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_MappingDialog</class>
 <widget class="QWidget" name="DB_MappingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Import Mapping</string>
  </property>
  <property name="windowIcon">
   <iconset resource="db.qrc">
    <normaloff>:/images/biluna16.png</normaloff>:/images/biluna16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="layoutWidget_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbHelp">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbBack">
       <property name="text">
        <string>&lt; &amp;Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbNext">
       <property name="text">
        <string>&amp;Next &gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageSettings">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>Import CSV file</string>
              </property>
              <layout class="QGridLayout" name="_5">
               <property name="leftMargin">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>6</number>
               </property>
               <property name="rightMargin">
                <number>6</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <property name="spacing">
                <number>6</number>
               </property>
               <item row="0" column="0">
                <widget class="QLineEdit" name="leCsv">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <layout class="QHBoxLayout" name="_6">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <spacer name="spacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pbOpenCsvFile">
                   <property name="text">
                    <string>&amp;Open</string>
                   </property>
                   <property name="autoDefault">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>Use mapping from database or file</string>
              </property>
              <layout class="QGridLayout" name="_2">
               <property name="leftMargin">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>6</number>
               </property>
               <property name="rightMargin">
                <number>6</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <property name="spacing">
                <number>6</number>
               </property>
               <item row="1" column="0">
                <layout class="QHBoxLayout" name="_3">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <spacer>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pbSelectMapping">
                   <property name="text">
                    <string>&amp;Select</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pbOpenMappingFile">
                   <property name="text">
                    <string>&amp;File</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="0">
                <widget class="QLineEdit" name="leMappingFile"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="layoutWidget_3">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Import into table</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="leSelectedTable">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="pbImportTable">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>24</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="arrowType">
                 <enum>Qt::LeftArrow</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>270</width>
                <height>13</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="DB_ConnectionWidget" name="connectionWidget">
           <property name="sizeIncrement">
            <size>
             <width>1</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>244</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget1">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>ID mapping settings</string>
              </property>
              <layout class="QGridLayout" name="_4">
               <property name="leftMargin">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>6</number>
               </property>
               <property name="rightMargin">
                <number>6</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <property name="spacing">
                <number>6</number>
               </property>
               <item row="1" column="1">
                <widget class="QLineEdit" name="leParentId">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Database parent ID</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QToolButton" name="pbParentId">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::LeftArrow</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="3">
                <widget class="QCheckBox" name="cbUuid">
                 <property name="text">
                  <string>Create new Uuid for ID field</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="spacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QTableView" name="tableView">
           <property name="sizeIncrement">
            <size>
             <width>1</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>244</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageMapping">
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="layoutWidget_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBoxHeaderRow">
           <property name="text">
            <string>File has header row</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>271</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbExportMapping">
           <property name="text">
            <string>E&amp;xport mapping</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSaveMapping">
           <property name="toolTip">
            <string>Save mapping to database</string>
           </property>
           <property name="text">
            <string>&amp;Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Table - File mapping </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageProgress">
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="3" column="0">
        <spacer name="spacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblStatus">
         <property name="text">
          <string>Please wait ...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Importing CSV file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DB_ConnectionWidget</class>
   <extends>QTreeView</extends>
   <header>db_connectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="db.qrc"/>
 </resources>
 <connections/>
</ui>
